include/database.py

```diff
@@ -32,3 +33,32 @@ def select(connectionString, sql, parameters):
     except psycopg2.DatabaseError as err:
         sys.stderr.write(f'A general database error occurred: {err}')
         raise DatabaseError
+
+
+# db connection to get studyId
```

Comment:
While this is true for in the case of what you are writing right now, remember that this function can be used by any insert statement that returns something, so avoid making reference to specific fields in favor of more general statements (e.g., "Execute the provided INSERT statement and return the result of the operation").